projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -805,20 +679,16 @@ class="<?php echo esc_attr( $classes['email_field'] ); ?> required"
 							<input type="hidden" name="sub-type" value="<?php echo esc_attr( $data['source'] ); ?>"/>
 							<input type="hidden" name="redirect_fragment" value="<?php echo esc_attr( $form_id ); ?>"/>
 							<?php
+							wp_nonce_field( 'blogsub_subscribe_' . $blog_id, '_wpnonce', false );
```

Comment:
That nonce was not present on the Jetpack form. Probably not an issue.

Reply:
Also, it used to be surrounded with:
```
<?php
							if ( is_user_logged_in() ) {
								wp_nonce_field( 'blogsub_subscribe_' . get_current_blog_id(), '_wpnonce', false );
							}
							?>
```
```suggestion
							if ( ( new Host() )->is_wpcom() && is_user_logged_in() ) {
							       wp_nonce_field( 'blogsub_subscribe_' . $blog_id, '_wpnonce', false );
							}
```

Reply:
It was surrounded by logged-in check in Jetpack, but not on WP.com:

https://github.com/Automattic/jetpack/blob/bba5b43f5f58d491403009a7816b4626973f8b3d/projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php#L695

https://github.com/Automattic/jetpack/blob/bba5b43f5f58d491403009a7816b4626973f8b3d/projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php#L817-L821

I'm fine with either way, as long as it's the same and works. 😁

Reply:
> It was surrounded by logged-in check in Jetpack, but not on WP.com:

Oh I missed that. I guess keeping it on both is fine, then...

Reply:
pushed [this commit](https://github.com/Automattic/jetpack/pull/33865/commits/31eba3884a6ffc1c893e7664afa5f2228e682e6d) so we always verify the nonce too